beetmoverscript/templates/mini_artifact_manifest.yml

```diff
@@ -0,0 +1,603 @@
+metadata:
+  description: Maps {{ product }} artifacts to pretty names
+  name: Beet Mover Manifest
+  owner: release@mozilla.com
+s3_bucket_paths:
+  devedition_candidates: pub/devedition/candidates/{{ version }}-candidates/build{{ build_number }}/
+  devedition_candidates_repacks: pub/devedition/candidates/{{ version }}-candidates/build{{ build_number }}/
+  fennec_candidates_repacks: pub/mobile/candidates/{{ version }}-candidates/build{{ build_number }}/
+  fennec_nightly: pub/mobile/nightly/
+  fennec_nightly_repacks: pub/mobile/nightly/
+  firefox_candidates: pub/firefox/candidates/{{ version }}-candidates/build{{ build_number }}/
+  firefox_candidates_repacks: pub/firefox/candidates/{{ version }}-candidates/build{{ build_number }}/
+  firefox_nightly: pub/firefox/nightly/
+  firefox_nightly_repacks: pub/firefox/nightly/
+  thunderbird_candidates: pub/thunderbird/candidates/{{ version }}-candidates/build{{ build_number }}/
+  thunderbird_candidates_repacks: pub/thunderbird/candidates/{{ version }}-candidates/build{{ build_number }}/
+  thunderbird_nightly: pub/thunderbird/nightly/
+  thunderbird_nightly_repacks: pub/thunderbird/nightly/
+mapping:
+  win32:
+    partials[partial].artifact_name:
+      default: partials[partial].artifact_name
+      devedition:
+        candidates:
+          pretty_name: update/{{ platform }}/{{ locale }}/firefox-{{ partials[partial].previousVersion }}-{{ version }}.partial.mar
+          pretty_path:
+          - update/{{ platform }}/{{ locale }}/firefox-{{ partials[partial].previousVersion }}-{{ version }}.partial.mar
+        candidates_repacks:
+          pretty_name: update/{{ platform }}/{{ locale }}/firefox-{{ partials[partial].previousVersion }}-{{ version }}.partial.mar
+          pretty_path:
+          - update/{{ platform }}/{{ locale }}/firefox-{{ partials[partial].previousVersion }}-{{ version }}.partial.mar
+        nightly:
+          pretty_name: ''
+          pretty_path: ''
+        nightly_repacks:
+          pretty_name: ''
+          pretty_path: ''
+        partner:
+          pretty_name: ''
+          pretty_path: ''
+        partner_repacks:
+          pretty_name: ''
+          pretty_path: ''
+      simple: partials[partial].artifact_name
```

Comment:
If `simple` never deviates from `default`, we could drop one.

Reply:
So far, this is just how I've designed it because I don't know how the Build team might want to use default - it may change